setup.py

```diff
@@ -15,11 +17,18 @@
 
 from setuptools import setup, find_packages
 
+# Grab description for Pypi
+with open('README.md') as fhl:
+    description = fhl.read()
+
```

Comment:
Ιt probably isn't valid to assume our `$PWD` is the same as the root of the repository; what if someone does:

```
cd ..
python ./iron-blogger2/setup.py ...
```

I think I've bumped up against that sort of thing when doing RPM packaging in the past. You probably need to specify